scripts/core/EXT.rst

```diff
@@ -0,0 +1,91 @@
+
+1. **Standard** -  extension will be included into the current and all future version of the specification.
+2. **Experimental** - extensions require additional experimentation and development, before becoming a standard extension. 
+Applications should not rely on experimental extensions in production.
+
+Requirements
+============
+
+- Extensions must use globally unique names for macros, enums, structures and functions
+- Extensions must have globally unique extension names reported from ${x}PlatformGetExtensionProperties
+- All extensions must be defined in this specification
+- Extensions must not break backwards compatibility of the core APIs
+- Standard extension versions must be backwards compatible with prior versions
+
+
+Naming Convention
+-----------------
+
+The following naming conventions must be followed for **standard** extensions:
+## --validate=off
+  - The extension name must begin with `${x}_ext`
+  - All extension functions must be postfixed with `Ext`
+  - All macros must use all caps `${X}_<NAME>_EXT` convention
+  - All structures, enumerations and other types must follow `${x}_<name>_ext_t` snake case convention
+  - All enumerator values must use all caps `${X}_ENUM_EXT_ETOR_NAME` convention
+  - All handle types must end with `ext_handle_t`
+  - All descriptor structures must end with `ext_desc_t`
+  - All property structures must end with `ext_properties_t`
+  - All flag enumerations must end with `ext_flags_t`
+## --validate=on
+
+The following naming conventions must be followed for **experimental** extensions:
+
+## --validate=off
+  - The extension name must begin with `${x}_exp`
+  - Experimental extensions may be added and removed from the driver at any time.
+  - Experimental extensions are not guaranteed to be forward- or backward-compatible between versions.
+  - Experimental extensions are not guaranteed to be supported in production driver releases; and may appear and disappear from release to release.
+  - All extension functions must be postfixed with `Exp`
+  - All macros must use all caps `${X}_<NAME>_EXP` convention
+  - All structures, enumerations and other types must follow `${x}_<name>_exp_t` snake case convention
+  - All enumerator values must use all caps `${X}_ENUM_EXP_ETOR_NAME` convention
+  - All handle types must end with `exp_handle_t`
+  - All descriptor structures must end with `exp_desc_t`
+  - All property structures must end with `exp_properties_t`
+  - All flag enumerations must end with `exp_flags_t`
+## --validate=on
+
+Extending Enumerations
+----------------------
+
+Any existing enumeration may be extended by adding new enumerators. Enumerators must use the extensions naming 
+convention and values should be assigned to avoid future compatibility issues.
+
+
+Extending Structures
+--------------------
+
+Any structure derived from `${x}_base_desc_t`` or `${x}_base_properties_t`` can be extended using structure chains. No other
+method of extending structures is allowed.
```

Comment:
Since we refer to these as `pNext` chains in the spec, we should do that here too.

Reply:
👍